imdb-ratings/Models/MovieGenreModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace imdb_ratings.Models
+{
+    public class GenreList
+    {
+        public IEnumerable<MovieGenreModel> Genres { get; set; }
+    }
+
+    public class MovieGenreModel
```

Comment:
```suggestion
    public class MovieGenre
```
Need not have model in there. It self explanatory based on the namespace it is in. having model in the name is redundant